packages/keyring-api/docs/sol-methods.md

```diff
@@ -0,0 +1,481 @@
+# SOL Methods
+
+Here we document the Solana methods that an account Snap may implement to
+support requests originated from dapps.
+
+## [solana:signAndSendTransaction](https://github.com/anza-xyz/wallet-standard/blob/master/packages/core/features/src/signAndSendTransaction.ts#L10)
+
+Signs and sends a transaction to the Solana blockchain.
+
+### [Parameters](https://github.com/anza-xyz/wallet-standard/blob/master/packages/core/features/src/signAndSendTransaction.ts#L42)
```

Comment:
Based on the wallet standard, how would we know the origin of the request?

Reply:
I was thinking this would be available in the request context, much in the same way it is EVM

Reply:
Added